content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

```diff
@@ -552,14 +552,12 @@ Retrieve a [Ticket object](ticket.html), which represents a [sequence number][]
 *Commandline*
 
 ```sh
-rippled json ledger_entry '{ "ticket": { "owner": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn", "ticket_sequence: 23 }, "ledger_index": "validated" }'
+rippled json ledger_entry '{ "ticket": { "account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn", "ticket_seq: 389 }, "ledger_index": "validated" }'
```

Comment:
Ah bugger, I forgot to mention, there's a missing ending quote for `ticket_seq`.
Change `"ticket_seq: 389` `"ticket_seq: 389"`.